auditlog/models.py

```diff
@@ -506,9 +507,11 @@ def changes_display_dict(self):
                     elif field_type in ["ForeignKey", "OneToOneField"]:
                         value = self._get_changes_display_for_fk_field(field, value)
 
-                    # check if length is longer than 140 and truncate with ellipsis
-                    if len(value) > 140:
-                        value = f"{value[:140]}..."
+                    if (
+                        settings.AUDITLOG_TRUNCATE_CHANGES_DISPLAY
+                        and len(value) > settings.AUDITLOG_TRUNCATE_LIMIT
+                    ):
+                        value = truncatechars(value, settings.AUDITLOG_TRUNCATE_LIMIT)
```

Comment:
as the logic is simple, I would prefer to keep it here instead of `Truncator` class